LICENSE.MPL-2.0

```diff
@@ -0,0 +1,385 @@
+eksctl makes use of the following pieces of code which are licensed under
+the MPL-2.0:
+
+  https://github.com/go-sql-driver/mysql
```

Comment:
Actually, we don't vendor this one. It maybe a sub-dependency, but we don't drag it along.